Find a functionally equivalent code:android.graphics.Paint.setMaskFilter:COMMENT
Method Modifier: public      
Comment:/**
 * Set or clear the maskfilter object.
 * <p />
 * Pass null to clear any previous maskfilter.
 * As a convenience, the parameter passed is also returned.
 *
 * @param maskfilter May be null. The maskfilter to be installed in the
 * paint
 * @return           maskfilter
 */

@@ -1,5 +1,5 @@
 {
-    int maskfilterNative = 0;
+    long maskfilterNative = 0;
     if (maskfilter != null) {
         maskfilterNative = maskfilter.native_instance;
     }

